tools/lkl/lib/hijack/hijack.c

```diff
@@ -283,3 +283,20 @@ int epoll_ctl(int epollfd, int op, int fd, struct epoll_event *event)
 
 	return lkl_call(__lkl__NR_epoll_ctl, 4, epollfd, op, fd, event);
 }
+
+int eventfd(unsigned int count, int flags)
+{
+	return lkl_sys_eventfd2(count, flags);
```

Comment:
btw setting a non-atomic variable is also not guaranteed to be atomic on some architectures.

From my POV the CHECK_HOST_CALL should be changed to something like:

if (!acquire_read(host_calls...)) {
        lock; local_var = dlsym...; unlock;
        release_write(host_calls..., local_var);
}